app/misc.py

```diff
@@ -0,0 +1,116 @@
+import json
+import math
+from pathlib import Path
+from typing import NamedTuple
+
+from app.customers.customer import Customer
+
+
+class Point(NamedTuple):
+    px: int
+    py: int
```

Comment:
Why you decided to make it px, py and not just x, y? Looks like just x, y will be more self-descriptive here